src/components/hooks.js

```diff
@@ -190,36 +190,32 @@ export function useGasPrice(enabled = true) {
       const run = async () => {
         const provider = await getProvider()
         const blockDetails = await provider.getBlock('latest')
-        const baseFeeWei = utils.formatUnits(blockDetails.baseFeePerGas, 'wei')
-        const price = {
-          slow: baseFeeWei + 2 * Math.pow(10, 9),
-          fast: baseFeeWei * 2 + 2 * Math.pow(10, 9)
+        if (blockDetails.baseFeePerGas) {
+          const baseFeeWei = utils.formatUnits(
+            blockDetails.baseFeePerGas,
+            'wei'
+          )
+          const price = {
+            slow: baseFeeWei + 2 * Math.pow(10, 9),
+            fast: baseFeeWei * 1.5 + 2 * Math.pow(10, 9)
+          }
+          setPrice(price)
+          setLoading(false)
+        } else {
+          const gasApi = 'https://www.gasnow.org/api/v3/gas/price'
```

Comment:
Could you move this constant to the very beginning of global scope? In case of URL change, it can be easier to find for others.

Reply:
I'd vote for defining constants at the top of the file instead of having a dedicated constants folder, just to keep it near to where it's being used